src/Main.java

```diff
@@ -0,0 +1,44 @@
+import java.util.Scanner;
+
+public class Main {
+    public static Scanner scanner = new Scanner(System.in);
+
+    public static void main(String[] args) {
+
+        while (true) {
+            System.out.println("******** Register *********");
+            System.out.println("please do choice");
+            System.out.println("1.Login 2.Register 3.exit 4.showMe list");
+            int choice = Integer.parseInt(scanner.nextLine());
+
+            switch (choice) {
+                case CommandConstants.LOGIN_CODE:
+                    System.out.println("Your name");
+                    String existName = scanner.nextLine();
```

Comment:
the same style everywhere
`name` -> `username`

Reply:
maybe better to use `input` instead of `exist` ?
It is a value that user inputs. It may not be in the system